examples/testdata/path_matcher/envoy_config.json

```diff
@@ -206,14 +206,10 @@
                         "rules": [
                           {
                             "operation": "1.examples_path_matcher_endpoints_cloudesf_testing_cloud_goog.GetShelf",
-                            "pathParameterExtraction": {
-                              "snakeToJsonSegments": {
-                                "shelf_id": "shelfId"
-                              }
-                            },
+                            "pathParameterExtraction": {},
```

Comment:
Please change path_matcher filter config to remove snake_to_json_segments field

Reply:
well,  this require change envoy c++ code.   let us change it, we don't want dead code

Reply:
It would be a lot to remove the dead code. I intend to put in another PR as I put in the description.